Cepheus-Class Mech Carrier! (CMM)

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This ship adds the Cepheus-Class Mech Carrier! 

![2023 01 13-13 19 22](https://user-images.githubusercontent.com/79304582/212318856-ffaa2b35-4240-4a5b-8630-73f9b0d38224.png)
____
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>

<summary>Image Dump</summary>

![image](https://user-images.githubusercontent.com/79304582/212320278-1a3547a6-d81a-4131-a0da-f4d58fc7a458.png)

![image](https://user-images.githubusercontent.com/79304582/212320336-3f42e9aa-6b9c-4926-bb49-2309873a9a60.png)

![image](https://user-images.githubusercontent.com/79304582/212320399-27504bab-17f1-4d86-9551-fffb8143c371.png)

![image](https://user-images.githubusercontent.com/79304582/212320492-681e78d8-2c33-496b-ac1b-8d29f393f579.png)

![image](https://user-images.githubusercontent.com/79304582/212320568-1ae54a7d-c217-4efc-bca6-336892d465ee.png)

![image](https://user-images.githubusercontent.com/79304582/212320613-16d41c2c-421a-45f4-96e5-7849fec4f886.png)

</details>

_____

it was first put into service a few years after the initial Xenomorph hives started cropping up om CMM worlds. While originally being made in response to Mech-equipped pirates, and exploration, it quickly got repurposed to fight the Xeno menace With its Sealed mechs. 

In recent years the need for the ship has declined with the production of specialized ships and better-equipped marines. Due to this, most of the vessels of its kind have fallen into disrepair, and are instead serving as stands in museums, or low-risk patrolling.

The in-game ship, while still in active duty, has fallen into slight disrepair, with Funds from central coming less and less frequently. This has led to the ship becoming mostly self-sufficient, relying on trading with other ships, and exploring/salvaging. 

the main equipment of the ship, would be the mechbay, and the machinery inside. They come equipped with a circuit imprinter, mech fabricator and a Security lathe, for printing weapons, and ammo for the Mechs. The armoury is also quite well stocked.
it does however in turn, lack quite a bit in medical equipment, and mining equipment (outside of mechs)

Crew:
1x Captain
2x Minuteman
2x Mech pilot
1x Roboticist
1x Engineer
3x cadet

Weapons: 3 m1911's / 1 cycler shotgun / captains Revolver (heavily limited on ammo)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
 While cmm's ship rooster has been increasing, the removal of the Carina left a vacant spot for a carrier-esque ship. This ship fills that niche!
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Cepheus class
add: Minutemen Mech pilot job
add: Minutemen Roboticist job
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
Main Files

Cep Json
cep DMM
Minutemen Roboticist
Minutemen Mech Pilot

spockye committed:
Update minutemen_cepheus.dmm

spockye committed:
Update minutemen_cepheus.dmm

spockye committed:
misc fixes

missing decals

(ultra important)  Missing bee

spockye committed:
adds cameras

Valorium-RB commented:
I signed off on this, lore-wise.

spockye committed:
Merge branch 'master' into minutemen_mech_small

Erikafox commented on _maps/shuttles/shiptest/minutemen_cepheus.dmm:
```diff
@@ -0,0 +1,5640 @@
+	},
+/obj/machinery/light,
+/obj/item/phone{
+	desc = "Supposedly a direct line to Nanotrasen Central Command. It's not even plugged in.";
```
Minutemen ship, Nanotrasen?

Erikafox reviewed (commented):
map review later

Apogee-dev commented:
Now that some more CMM content has been merged, could you go ahead and replace relevant items with their new CMM equivalents? examples would be the cycler shotgun -> CM-15, ripley -> CMM ripley, and so on.

spockye committed:
Merge remote-tracking branch 'upstream/master' into minutemen_mech_small

spockye committed:
Update security_officer.dm

spockye committed:
misc fixes

changes the main minutemen job to have the new backpack

switched the cycler for the new CMM shotgun

fixes phone desc

switches out mechs

Erikafox reviewed (commented):
looks good for tm

Erikafox reviewed (changes requested):
Some doors should get names

The ship needs to not fly sideways

spockye committed:
Merge branch 'master' into minutemen_mech_small

spockye committed:
adds orm board adds ore box

spockye committed:
Merge branch 'minutemen_mech_small' of https://github.com/spockye/Shiptest into minutemen_mech_small

spockye committed:
fixes fucky made in previous commit

spockye committed:
Merge branch 'master' into minutemen_mech_small

spockye committed:
gives all the doors names

spockye committed:
Merge branch 'minutemen_mech_small' of https://github.com/spockye/Shiptest into minutemen_mech_small

spockye committed:
Merge branch 'master' into minutemen_mech_small

spockye committed:
Merge branch 'master' into minutemen_mech_small

spockye committed:
Merge branch 'minutemen_mech_small' of https://github.com/spockye/Shiptest into minutemen_mech_small

spockye committed:
misc fixes

fixes door names
attempts to fix flight dir

spockye committed:
Update minutemen_cepheus.dmm

spockye committed:
Update minutemen_cepheus.dmm

spockye committed:
Update minutemen_cepheus.dmm

Erikafox commented:
Did you fix it flying sideways

Erikafox commented:
once it doesn't fly sideways and has a mech disk it's good to go.

spockye committed:
fixes rotation

spockye committed:
Merge remote-tracking branch 'upstream/master' into minutemen_mech_small

spockye committed:
adds a CMM-Mech design disk

Erikafox commented:
I vaguely recall seeing it fly the right direction yesterday.
